src/index.ts

```diff
@@ -32,23 +32,23 @@ export type RainLanguageServicesOptions = {
  */
 export class RainlangCodemirror {
     private plugin: RainLanguageServicesPlugin | undefined;
-    public extension: Extension[] = [];
```

Comment:
@marcusnewton this is specifically named `extension`, so that the object class can be used directly as an extension